src/index.js

```diff
@@ -48,7 +48,7 @@ class ReactIntlUniversal {
       warningHandler: console.warn, // ability to accumulate missing messages using third party services like Sentry
       escapeHtml: true, // disable escape html in variable mode
       commonLocaleDataUrls: COMMON_LOCALE_DATA_URLS,
-      fallbackLocale: null, // Locale to use if a key is not found in the current locale
+      fallbackLocale: null, // Locales to use if a key is not found in the current locale, such as 'zh-CN;en' will use the key in locale 'zh-CN', if the specific key not exist in 'zh-CN', will fallback to 'en'
```

Comment:
Should we add another parameters `fallbackLocales` with type of `string[]`? And make the original `fallbackLocale` deprecated.

Reply:
Hi @cwtuan 
Thanks a lot for your great advice. Here attached the modification according to your suggestion, please have a kind review again.